packages/server/src/database/migrations/mysql/1712762777471-ExpandChatflowConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class ExpandChatFlowConfig1712762777471 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE \`chat_flow\` MODIFY \`chatbotConfig\` LONGTEXT;`)
```

Comment:
is it `chatbotConfig` that should be changed? isnt it `flowData`?

Reply:
I think that’s May purpose of prevention of exceptions.